deploy/docs/Installation_with_Helm.md

```diff
@@ -0,0 +1,98 @@
+# Installation with Helm
+
+Our Helm chart deploys Kubernetes resources for collecting Kubernetes logs, metrics, and events; enriching them with deployment, pod, and service level metadata; and sends them to Sumo Logic.
+
+<!-- TOC -->
+ 
+- [Prerequisite](#prerequisite)
+- [Installation Steps](#installation-steps) 
+- [Uninstalling the Chart](#uninstalling-the-chart) 
+
+<!-- /TOC -->
+
+## Prerequisite
+
+Sumo Logic Apps for Kubernetes and Explore require you to add the following [fields](https://help.sumologic.com/Manage/Fields) in the Sumo Logic UI to your Fields table schema. This is to ensure your logs are tagged with relevant metadata.
+- cluster
+- container
+- deployment
+- host
+- namespace
+- node
+- pod
+- service
+
+The Helm chart installation requires three parameter overrides:
```

Comment:
Can we document the clusterName and externalLabels here or at least before the install steps? It is always missed